ozaria/site/components/teacher-dashboard/modals/ModalEditClass.vue

```diff
@@ -142,6 +142,9 @@ export default Vue.extend({
       return ((this.classroom || {}).aceConfig || {}).codeFormatDefault || 'text-code'
     },
     levelChat () {
+      if (utils.isCodeCombat) {
```

Comment:
why are we changing computed property value? We should just set value in `data()` with a default value like you did with language. If it's an existing class, we don't want to enable it automatically and reflect whatever the current settings are. Only new classes should have this default value.